modules/enhancer.js

```diff
@@ -48,30 +48,15 @@ var enhanceWithRadium = function (ComposedComponent: constructor): constructor {
   // with IE <10 any static properties of the superclass aren't inherited and
   // so need to be manually populated
   // See http://babeljs.io/docs/advanced/caveats/#classes-10-and-below-
-  var staticKeys = [
-    'defaultProps',
-    'propTypes',
-    'contextTypes',
-    'childContextTypes'
-  ];
-
-  staticKeys.forEach((key) => {
-    if (ComposedComponent.hasOwnProperty(key)) {
-      RadiumEnhancer[key] = ComposedComponent[key];
+  // This also fixes React Hot Loader by exposing the original components top level
+  // prototype methods on the Radium enhanced prototype as discussed in #219.
+  Object.keys(ComposedComponent.prototype).forEach(key => {
```

Comment:
ES6 class properties are not enumerable with `Object.keys`, as per the spec. They are defined with `Object.defineProperty` where `enumerable` is false by default. So the keys will not show up using `Object.keys()` and will not be copied. When you assign a vanilla property with `MyComponent.foo = someVal`, it will show up because the old default for `enumerable` when not using `Object.defineProperty` is true.